Library/Homebrew/os/mac.rb

```diff
@@ -41,6 +41,10 @@ def cat
       version.to_sym
     end
 
+    def language
+      @language ||= `defaults read .GlobalPreferences AppleLanguages`.delete(" \n\"()").sub(%r{,.*}, '')
```

Comment:
Feels weird on hindsight to return a comma-separated string. `.delete("()\"\n ").split(",")` seems to do what you want and return an array.